k9mail-library/src/main/java/com/fsck/k9/mail/internet/MessageExtractor.java

```diff
@@ -193,7 +194,7 @@ public static void findViewablesAndAttachments(Part part,
             Viewable viewable;
             if (isSameMimeType(mimeType, "text/plain")) {
                 if (isFormatFlowed(part.getContentType())) {
-                    viewable = new Flowed(part);
+                    viewable = new Flowed(part,isDelSp(part.getContentType()));
```

Comment:
Finding out whether the `delsp` parameter is present can be done inside the `Flowed` constructor.